CONTRIBUTING.md

```diff
@@ -12,15 +12,16 @@ Contributions usually come in the form of enhancements to Postman files or docum
 
 </br>
 
+
 ### Make a **Postman** pull request
 
 > Make your pull requests for contributions to the collection **in the Postman environment**, not on this repo!
 
-The steps are very similar to making a pull request in Git:
+The basic steps are very similar to making a pull request in Git:
 
-1. [Create a fork of the collection](https://www.postman.com/salesforce-developers/workspace/salesforce-developers/collection/12721794-7d783742-165f-4d10-8c4c-5719fb60fba2/fork?origin=sidebar) in Postman.
+1. [Create a fork of the collection](https://www.postman.com/salesforce-developers/workspace/salesforce-developers/collection/12721794-7d783742-165f-4d10-8c4c-5719fb60fba2/fork?origin=sidebar) in Postman. You may also need to fork the environment variables file for this collection.
```

Comment:
When I read this, I want to know what steps are needed, but the word "may" is not comforting me. When should I do that and when not?